.github/workflows/content.yml

```diff
@@ -44,6 +44,20 @@ jobs:
         steps:
             - name: Checkout 🛎️
               uses: actions/checkout@v2.3.1
+              
+            - name: fetch git tags
+              id: get_tag_name
+              run: |
+                   git fetch --prune --unshallow --tags
+                   GIT_TAG_NAME=`git tag --format='%(creatordate:short)%09%(refname:strip=2)' --sort=creatordate | grep production | awk '{ print $2; }' | tail -1`
```

Comment:
lets not use `creatordate` for sorting instead use `taggerdate`. can we please reproduce fetching git tag by using this `git for-each-ref --sort=taggerdate --format '%(tag)' | grep production | tail -1` ?

Reply:
`tail -10` is basically reducing grepping over along list of tags.

Reply:
LGTM